src/components/Powers.jsx

```diff
@@ -64,6 +63,20 @@ const Powers = (props) => {
       {values.contract.power_type === 'power-lower-15kw' ? (
         <Grid item xs={12}>
           <Grid container spacing={2}>
+            <Grid item>
+              <HelperText
+                helperText={
+                  <a
```

Comment:
I think there is an agreement to use MUI material components instead of html raw tags. I don't think it has to change, and maybe there is a problem used as `helperText`. Not a must, just a comment. :call_me_hand: